core/math/expression.h

```diff
@@ -76,6 +76,7 @@ class Expression : public Reference {
 		MATH_RAND,
 		MATH_RANDF,
 		MATH_RANDOM,
+		MATH_RANDOM_INT,
```

Comment:
While we're at it, maybe those four enum values should be improved too? (Same in the duplicates in GDScript and VisualScript.

The current names are not very clear and there's a mismatch between the use of `int` and `float` qualifiers.

How about:
```
MATH_RANDI
MATH_RANDF
MATH_RANDI_RANGE
MATH_RANDF_RANGE
```
like the methods?

Reply:
Yes, good idea, renamed